packages/discovery-provider/src/queries/get_disbursed_challenges_amount.py

```diff
@@ -20,4 +21,13 @@ def get_disbursed_challenges_amount(
         )
         .scalar()
     )
-    return amount_disbursed
+    if not amount_disbursed:
+        return 0
+    return amount_disbursed / 10**WAUDIO_DECIMALS
+
+
+def get_weekly_pool_window_start(now):
```

Comment:
am confused, why is this helper fn in this file if it's only used in `get_attestation.py`?

Reply:
it's used in the new endpoint /challenge_id/info as well. I figure it has more to do with "disbursing" a challenge versus getting an attestation for one. lmk if you want it elsewhere!